syntax/blame.sublime-settings

```diff
@@ -0,0 +1,16 @@
+{
+	"always_show_minimap_viewport": false,
```

Comment:
This does seem like a good idea. We should discuss each setting. I would not like to set more and we can confirm is good, the reason for this is that the user can set this in his/her user settings.

Reply:
This setting makes most sense for output panel syntaxes, propably.